sympy/ntheory/generate.py

```diff
@@ -692,6 +696,7 @@ def primerange(a, b):
         See Also
         ========
 
+        prime : Return the ith prime
         nextprime : Return the ith prime greater than n
```

Comment:
What does `ith prime greater than n` mean?

Reply:
It should read "return the next prime greater than i"